source/hardware/controllers/denon_mc7000.rst

```diff
@@ -296,6 +319,7 @@ Use :hwlabel:`SHIFT` + Pad button to stop a sampler while playing or eject a sam
    :alt: Denon MC7000 SAMPLER Mode
    :figclass: pretty-figures
 
+.. _LEDs:
```

Comment:
These labels are global, let's make it more descriptive:

```suggestion
.. _denon-mc7000-leds:
```

Reply:
Good point. I gonna change that. Thanks

Reply:
all updated to reference locally ...